### PR TITLE
remove worker.join in __response_check_target, to fix bug where register...

### PR DESCRIPTION
**Pull request by anwar6953**
...ing callbacks was blocking. remove test case for file-send (feature was removed). config.json->default-config.json for consistency.

### CODE REVIEW

**Comment by rp-julienguery** [comment]
LGTM

File: cdap-stream-clients/python/streamclient/v2/streampromise.py [hunk 1]
```diff
@@ -126,26 +128,29 @@ def __worker_target(self, uri, dataDict):
...
-
-                if self.__onOkHandler:
-                    self.__onOkHandler(self.__serviceResponse)
-            except NotFoundError:
-                if self.__onErrorHandler:
-                    self.__onErrorHandler(self.__serviceResponse)
-            finally:
-                self.__onOkHandler = self.__onErrorHandler = None
+            funcList = []
+            if self.__HTTP_OK is self.__serviceResponse.status_code:
```
Comment (ParthGandhi) [inline cdap-stream-clients/python/streamclient/v2/streampromise.py]: Replace "is" with "==".

File: cdap-stream-clients/python/streamclient/v3/streampromise.py [hunk 2]
```diff
@@ -127,25 +130,28 @@ def __worker_target(self, uri, dataDict):
...
-
-                if self.__onOkHandler:
-                    self.__onOkHandler(self.__serviceResponse)
-            except NotFoundError:
-                if self.__onErrorHandler:
-                    self.__onErrorHandler(self.__serviceResponse)
-            finally:
-                self.__onOkHandler = self.__onErrorHandler = None
+            funcList = []
+            if self.__HTTP_OK is self.__serviceResponse.status_code:
```
Comment (ParthGandhi) [inline cdap-stream-clients/python/streamclient/v3/streampromise.py]: "=="

**Comment by ParthGandhi** [comment]
One comment. Fix and merge.

**Comment by ParthGandhi** [comment]
LGTM.

**Pull request merged**